### ISSUE TITLE
NullPointerException

### ISSUE THREAD

**Issue by Robertbaelde** [bitpay/cordova-plugin-qrscanner#101]
First of all, thanks for the great plugin. 

In the play store dashboard, the following crash reports keep showing up. Is this a bug in the implementation or in the plugin? 

java.lang.NullPointerException: 
  at com.bitpay.cordova.qrscanner.QRScanner$19.run(QRScanner.java:546)
  at android.app.Activity.runOnUiThread(Activity.java:5562)
  at com.bitpay.cordova.qrscanner.QRScanner.prepare(QRScanner.java:543)
  at com.bitpay.cordova.qrscanner.QRScanner.access$1500(QRScanner.java:38)
  at com.bitpay.cordova.qrscanner.QRScanner$11$1.run(QRScanner.java:192)
  at android.os.Handler.handleCallback(Handler.java:739)
  at android.os.Handler.dispatchMessage(Handler.java:95)
  at android.os.Looper.loop(Looper.java:152)
  at android.app.ActivityThread.main(ActivityThread.java:5497)
  at java.lang.reflect.Method.invoke(Native Method:0)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)

**Comment by ivan94fi** [comment]
Yes i'm experiencing this error too. It happens when i tap the scan button on my app. Sometimes it works fine, sometimes the app force closes (several times in a row before returning to work) and these errors appear on my logcat.
I "wrote a workaround", just to prevent the force close: a simple control on that mBarcodeView object to verify if it's null. If it actually is, i signal it in the logcat and then return. Here's the code i added in 

`<path-to-my-cordova-app>\platforms\android\src\com\bitpay\cordova\qrscanner\QRScanner.java`

First of all include Log class, used to write to logcat:
`import android.util.Log;`

Then add the `if(mBarcodeView == null) {...}` at around line 544, just before `mBarcodeView.pause();`


Here's some context:

```java
other code...
else {
    prepared = false;
    this.cordova.getActivity().runOnUiThread(new Runnable() {
        @Override
        public void run() {
            /**********************************************/  
            if(mBarcodeView == null) {                                 //  <= I added this if
                Log.e("MyFirenze", "Error: mBarcodeView is null. Returning");
                return;
            }
            /************************************************/
            mBarcodeView.pause();          // <= The error happens here, since mBarcodeView might be null
        }
    });
more code...
```

Hope this helps someone and also hope that the dev will address this issue.

**Comment by bitjson** [comment]
Thanks for the report! This may be solved in [2.6.0](https://github.com/bitpay/cordova-plugin-qrscanner/blob/master/CHANGELOG.md#260-2018-05-17). If you're still having trouble, please open a new issue or send a PR 🚀
